cmd/dbc/main.go

```diff
@@ -134,6 +134,7 @@ type cmds struct {
 	Info       *InfoCmd         `arg:"subcommand" help:"Get information about a driver"`
 	Docs       *DocsCmd         `arg:"subcommand" help:"Open driver documentation in a web browser"`
 	Remove     *RemoveCmd       `arg:"subcommand" help:"Remove a driver from the driver list"`
+	Auth       *AuthCmd         `arg:"subcommand" help:"Manage driver index authentication credentials"`
```

Comment:
```suggestion
	Auth       *AuthCmd         `arg:"subcommand" help:"Manage driver registry credentials"`
```

I think we can drop "authentication" and just say "credentials" since it is implied